CONTRIBUTING.md

```diff
@@ -52,9 +53,13 @@ If you find a bug in Gravitino, please open an issue on GitHub. Be sure to inclu
 
 If you have ideas for enhancements or new features, feel free to create an issue to discuss them. Gravitino welcomes suggestions and provides prompt feedback on their feasibility and relevance.
 
+### Good first issues
+
+If you are new to open source or can't find something to work on check out the [Good First Issues list](https://github.com/datastrato/gravitino/contribute).
```

Comment:
Should we use `https://github.com/datastrato/gravitino/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22`?

Reply:
No https://github.com/datastrato/gravitino/contribute is a better link and lists all good first issues.

Reply:
It also links to the contribution guide.

Reply:
OK, I just clicked the link. It went wrong. But I clicked again. It's ok now.